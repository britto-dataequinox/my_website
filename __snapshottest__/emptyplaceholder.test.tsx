import React from 'react';
import renderer from 'react-test-renderer';
import EmptyPlaceholder from '../components/Placeholder/emptyPlaceholder';

test('empty placeholder snap', () => {
  const ChildComponent = () => <div>Child Component</div>;
  const tree = renderer
    .create(
      <EmptyPlaceholder>
        <ChildComponent />
      </EmptyPlaceholder>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
