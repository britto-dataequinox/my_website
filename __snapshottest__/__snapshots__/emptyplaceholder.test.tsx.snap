// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty placeholder snap 1`] = `
<div
  className="MuiBox-root css-ww7h5q"
>
  <div>
    Child Component
  </div>
</div>
`;
